src/Configuration/Http/Requests/ConfigureRequest.php

```diff
@@ -4,7 +4,7 @@
 
 namespace MyParcelCom\Integration\Configuration\Http\Requests;
 
-use Illuminate\Foundation\Http\FormRequest;
+use MyParcelCom\Integration\Http\Requests\FormRequest;
```

Comment:
Is there a reason to do that? If it's because of the shopId - the generic FormRequest is for routes that still have the shop_id passed as a query parameter. New endpoints have the param in the route. I'd suggest we dismiss this PR.

Reply:
oh okay, didn't know what was new and what was old, but I did notice the difference between query param and route param. sure, let's close thise